client/verta/tests/unit_tests/test_endpoint.py

```diff
@@ -69,212 +69,238 @@ def mock_endpoint(mock_conn, mock_config) -> Endpoint:
 GET_ACCESS_TOKEN_RESPONSE: str = "123-test-456-token-789"
```

Comment:
Make sure to review with "Hide whitespace" turned on.

Reply:
Thank you for this.  Did not know about that option